index.html

```diff
@@ -3,89 +3,81 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/style.css">
-    <title>Book store!</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Books For All</title>
 </head>
 <body>
-    
-<div class="container">
+   
+ <div class="container">
 
+   <div class="top-section">
     <h1>Books for all!</h1>
-    <div class="banner">
-        <div class="titlebanner">The home for your favourite books!</div>
-        <div>
-            <button class="btn">READ MORE</button>
-        </div>
+   </div>  
+
+   <div class="img-section">
```

Comment:
This is a banner, not an image, and image is maybe to generic. Class name doesn't add any context.